src/main/java/com/moodybluez/enterprise/dao/EntryDAOStub.java

```diff
@@ -38,6 +44,11 @@ public Entry save(Entry entry) {
     }
 
     @Override
```

Comment:
try and keep the annotations below the javadoc and inline with the methods so they are easier to read